#This message is to get desired position avoiding the usage of quaternions
std_msgs/Float64 x
std_msgs/Float64 y
std_msgs/Float64 theta #equivalent to yaw
